concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1402,7 +1527,11 @@ jobs:
         - task: sync-admin-users
           config:
             platform: linux
-            image: docker:///ruby#2.2
+            image_resource:
+              type: docker-image
+              source:
+                repository: ruby
+                tag: 2.2-slim
```

Comment:
This changed the container from `2.2` to `2.2-slim`. This has led to errors running the sync-admin-users script as it can no longer `bundle install` because the slim container doesn't include a compiler.

I'll do a PR to fix this.